python_ta/util/tree.py

```diff
@@ -0,0 +1,20 @@
+"""
+Simple Tree class to be used for RecursionTable
+"""
+
+
+class Tree:
+    def __init__(self, value):
+        self.value = value
+        self.children = []
+
+    def add_child(self, child_node):
+        self.children.append(child_node)
+
+    def check_tree_equality(self, tree1):
```

Comment:
Name this method `__eq__` and then in the tests below use `==` to check whether the trees are equal.